src/python_testing/mdns_discovery/enums/mdns_service_type.py

```diff
@@ -0,0 +1,28 @@
+#
+#    Copyright (c) 2025 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+from enum import Enum
+
+
+class MdnsServiceType(Enum):
+    """
+    Enum for Matter mDNS service types used in network service discovery.
+    """
+    COMMISSIONER = "_matterd._udp.local."
+    COMMISSIONABLE = "_matterc._udp.local."
+    OPERATIONAL = "_matter._tcp.local."
+    BORDER_ROUTER = "_meshcop._udp.local."
```

Comment:
Do you use this anywhere?

Reply:
specifically I meant the border router - are you doing anything with the border router?